components/codeflare/codeflare.go

```diff
@@ -90,11 +99,8 @@ func (c *CodeFlare) ReconcileComponent(ctx context.Context,
 				dependentOperator, ComponentName)
 		}
 
-		// Update image parameters only when we do not have customized manifests set
-		if (dscispec.DevFlags == nil || dscispec.DevFlags.ManifestsUri == "") && (c.DevFlags == nil || len(c.DevFlags.Manifests) == 0) {
-			if err := deploy.ApplyParams(ParamsPath, imageParamMap, map[string]string{"namespace": dscispec.ApplicationsNamespace}); err != nil {
-				return fmt.Errorf("failed update image from %s : %w", CodeflarePath+"/bases", err)
-			}
+		if err := deploy.ApplyParams(ParamsPath, map[string]string{}, map[string]string{"namespace": dscispec.ApplicationsNamespace}); err != nil {
```

Comment:
```suggestion
		if err := deploy.ApplyParams(ParamsPath, nil, map[string]string{"namespace": dscispec.ApplicationsNamespace}); err != nil {
```
do not even need an empty map ?  same for the ray one?

Reply:
yeah, leftover. (It's a bit counter intuitive that access nil map does not crash)